sklearn/isotonic.py

```diff
@@ -190,6 +190,12 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     X_max_ : float
         Maximum value of input array `X_` for right bound.
 
+    interpolation_X_ : array-like of shape (n_interpolation_points,)
```

Comment:
I would rather rename those attributes to `X_thresholds_` and `y_thresholds_` and explain that they include the list of de-duplicated ascending X and y values suitable to interpolate the  `y = f(X)` monotonic function.